Experimental/svf2

AlexPiro opened the pull request:
Experiments on SVF2 based on your work with OTG. Most part of the client / reader / parser are identical. I added the IMF part for SVF2 (as was done for SVF), but I think some of the logic should be done during the parsing.

My code is a little messy, let me know if you want me to clean it before pull request. 😉

petrbroz commented:
Hi Alex, yeah, this PR looks quite scary :D If you could clean it up, that'd certainly help.

AlexPiro commented:
I added a new branch from `develop` and cleaned a little my code, but I can't pull request on a new branch. Can you create a new one that I can pull request on ?
